scripts/setup/requirements.nrfconnect.txt

```diff
@@ -1,6 +1,7 @@
+-r ../../src/setup_payload/python/requirements.txt
```

Comment:
I think we have to re-think this: our bootstrap logic in CI creates caches of environments and to determine if it is up to date or not, it does a hash of `scripts/setup` and `third_party/pigweed`: https://github.com/project-chip/connectedhomeip/blob/master/.github/actions/bootstrap-cache/action.yaml#L14

This means that if we do this redirection, we would not detect that python environment changed between cache runs if requirements.txt changes from src.

As ugly as this may be, it may be better to copy these over instead, maybe with a comment (in both files?) that these have to be in sync.

Reply:
or move `setup_payload/python/requirements.txt` to `scripts/setup/requirements.setuppayload.txt` and include then include that in `requirements.nrfconnect.txt`?

Reply:
Ok, I will align the changes. Thanks.